### PR TITLE
Location in events.info should be a string and not an address

### PR DESCRIPTION
**Pull request by regularjack**
Location in `/events.info` should be a string and not an address. It already is a string in `/events.list`

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1417,7 +1417,7 @@ Get details for a single calendar event.
                 + id: `28c969df-a392-48f2-8540-c083e57957e4` (string)
             + starts_at: `2016-02-04T16:00:00+00:00` (string)
             + ends_at: `2016-02-04T18:00:00+00:00` (string)
-            + location (Address)
+            + location (string)
```
Comment (mrnnn) [inline apiary.apib]: We just copy the entire address into a string? Is that what is happening here @regularjack ? 

Something similar is happening in invoices and it wasn't pretty. So not sure we should actually do this or migrate to Address (and maybe even leave out of scope for now). @jenssegers

Comment (regularjack) [reply apiary.apib]: Yeah, this is a tough one. It was discussed with @stivni and @jenssegers. So, the problem is that with CalDAV, location is a string. So, CalDAV clients just send a string which means events will be created with an unstructured address. I think for "the future" there's no way around that, event locations will be strings.

That said, if we're not comfortable making this decision now, I'm ok with dropping location for now.

Comment (stivni) [reply apiary.apib]: well, @mrnnn, look at it like this: an event location can be "HQ, King's landing"

all calendars work this way...

Comment (mrnnn) [reply apiary.apib]: Yep, understand that now. Was just worried that we'd do a lossy conversion here, like is the case in invoicing. but the caldav context makes sense.

 :ok: by @mrnnn on 01/12/2017 10:04:09

**Pull request merged**